cli/src/commands/run_config_docker.py

```diff
@@ -12,35 +13,70 @@ def get_usage(self):
         return super(RunConfigDocker, self).get_usage() + " config"
 
     def get_options(self):
-        return [{"name": "config", "description": "path to the config to run"}]
+        return [{"name": "config",
+                 "description": "path to the config to run from the remote docker image"},
+                {"name": "from_local_code",
+                 "description": "use the local code to run the crawl",
+                 'optional': False}]
+
+    @staticmethod
+    def from_local_code_parse(args):
+        print(args)
+        if len(args) < 2:
+            return False
+        if isinstance(args[1], bool):
+            return args[1]
+        return isinstance(args[1], str) and args[1].lower() == 'true'
 
     def run(self, args):
-        from os import getcwd, environ
 
```

Comment:
Could be interesting to name the arguments, to make the code more readable. Something like this for example:
```python
config_path = args[0];
```

And then using `config_path` instead of `args[0]`. It makes skimming over the code easier as the variable expresses what it it already.